### ISSUE TITLE
Move Python `font.genericGlyphChange` description to `selection` sub-section

### ISSUE THREAD

**Issue by skef** [fontforge/fontforge#4131]
There are a few open issues referencing the lack of a Python interface for "Change Glyph". These stem from:

1. The somewhat different name (`genericGlyphChange`) of the python method (which traces to the C function names)
2. The lack of a glyph-level equivalent
3. The fact that `genericGlyphChange()` is documented in the wrong section of `python.html`. It is, sensibly, a selection-based interface but is listed among the regular `font` methods. 

I've been meaning to fix 2, which should actually be pretty easy: just break out the parameter processing into a separate method called in two places. (There's already a C-level `CVGenericChange()` to match `FVGenericChange()`. #2925 represents that work. 1 is easy enough to document around. I'm filing this to represent 3. Normally I would just go in and fix it but with #4113 outstanding I suspect that parallel merges are likely counterproductive right now.

In short, the task is to move the description in the `python.html` to the correct location. 

Happily the interface itself is keyword based and quite appropriate as-is. And anyone wanting to alter an individual glyph can currently just select it beforehand. It's a shame few people are aware of it.

**Comment by skef** [comment]
I guess the move would also be a good time to do something about problem 1, which can be as simple as mentioning something about "Change Weight" somewhere in column 1 or 3 (or their equivalents).

**Comment by skef** [comment]
Once #4133 is integrated the docs section should be copied to the glyph object. (Or, better, the glyph doc should have a subset of the info and refer to the selection entry, or vice versa, as with the various incarnations of `stroke`.)
